Zend/zend_lazy_objects.c

```diff
@@ -806,7 +806,11 @@ zend_property_info *zend_lazy_object_get_property_info_for_slot(zend_object *obj
 	zend_property_info **table = obj->ce->properties_info_table;
 	intptr_t prop_num = slot - obj->properties_table;
 	if (prop_num >= 0 && prop_num < obj->ce->default_properties_count) {
-		return table[prop_num];
+		if (table[prop_num]) {
```

Comment:
When ``table[prop_num]`` may be NULL?
Only for regular properties overridden with hooked ones?

Reply:
>  Only for regular properties overridden with hooked ones?

They will only ever be `NULL` for properties with hooks that override properties without hooks. This could add an `EXPECTED()` guard, if you like.

Reply:
OK. I don't care.